resources/qml/PrinterSelector/MachineSelectorButton.qml

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) 2018 Ultimaker B.V.
+// Cura is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.7
+import QtQuick.Controls 2.1
+
+import UM 1.1 as UM
+import Cura 1.0 as Cura
+
+Button
+{
+    id: machineSelectorButton
+
+    width: parent.width
+    height: UM.Theme.getSize("action_button").height
+    leftPadding: Math.round(1.5 * UM.Theme.getSize("default_margin").width)
+    rightPadding: Math.round(1.5 * UM.Theme.getSize("default_margin").width)
+    checkable: true
+
+    property var outputDevice: null
+    property var printerTypesList: []
+
+    function setPrinterTypesList()
```

Comment:
```suggestion
    function updatePrinterTypesList()
```
We tend to use set for actual setters, but that's not what this function does.

Reply:
Done in fa1ef5c45c70ebdebaef3d9d19b026e6586523b2